app/src/main/java/com/pokedex/koto/pokedex/ListResponse.java

```diff
@@ -6,37 +6,13 @@
 
 import java.util.List;
 
-import javax.xml.transform.Result;
-
 public class ListResponse {
 
     private int count;
-    private Object previous;
-    private List<Result> results = null;
+    private String previous;
```

Comment:
Changed `previous` type to `String` as it is the representation of previous URL when using pagination.